src/table/table-role/utils.ts

```diff
@@ -33,62 +28,61 @@ export function findFocusinCell(event: FocusEvent): null | FocusedCell {
   }
 
   const cellFocusables = getFocusables(cellElement);
-  const elementIndex = cellFocusables.indexOf(element);
+  const elementIndex = cellFocusables.indexOf(focusedElement);
 
-  return { rowIndex, colIndex, rowElement, cellElement, element, elementIndex };
+  return { rowIndex, colIndex, rowElement, cellElement, element: focusedElement, elementIndex };
```

Comment:
Why do we return the element here if its the same as the focusedElement provided?